### ISSUE TITLE
A question for distilling data2vec

### ISSUE THREAD

**Issue by Ther-nullptr** [s3prl/s3prl#378]
Hey! I want to distill a [data2vec](https://arxiv.org/abs/2202.03555) model(which is from torch.hub), following the steps of [distilling hubert](http://arxiv.org/abs/2110.01900). 
According to [README.md](https://github.com/s3prl/s3prl/blob/master/s3prl/pretrain/README.md), I only have to change the teacher model's name. However, I only get this result below when distilling the data2vec:

![0a79c2f9a6420edd8ff1e842cf8a3e7](https://user-images.githubusercontent.com/71265304/178115727-159ed9a5-ba47-48e7-9214-ddab03bed728.png)

Does this mean that the interface of distilling is not suitable for data2vec?

**Comment by vectominist** [comment]
During distillation of hubert, this phenomenon occurs seldomly.
I guess some reasons for this problem include
- Using `fp16`
- The norm of hidden representations of data2vec might be too large to distill (you may try inspecting this part and probably require some normalization beforehand)

**Comment by Ther-nullptr** [comment]
I disabled the fp16 training, and this phenomenon disappeared. Thank you!
